### DISCUSSION TITLE
v1.16.1

### DISCUSSION THREAD

**Discussion by Victor-IX** [Victor-IX/Blender-Launcher-V2#16] [Announcements]
# What's Changed
Hot Fix patch
## ⚠Info:
- **Blender Foundation is currently blocking the request on the /release website for the stables versions making it impossible to retrieve those versions, this issue will be fixed during the week**

## 🛠️Fix:
- Fix a bug where .msi version where downloaded instead of the .zip one

<hr /><em>This discussion was created from the release <a href='https://github.com/Victor-IX/Blender-Launcher-V2/releases/tag/v1.16.1'>v1.16.1</a>.</em>

**Comment by eisenbruch** [comment]
Hey yall, any update on getting stable build download stuff working for an upcoming release?
